README.md

```diff
@@ -1 +1,29 @@
-# engine-deploy-local
\ No newline at end of file
+# thirdweb Engine Deploy Locally
+
+Let's deploy thirdweb Engine locally in easiest way possible. This repository is maintained by thirdweb community team.
+
+> [!WARNING]
+> Use this for demo / preview purposes only. This is not recommended for production use. If you wanna use thirdweb Engine in production, try [cloud-hosted version](https://thirdweb.com/dashboard/engine?requestCloudHosted) or deploy a [self-hosted version](https://blog.warengonzaga.com/how-to-deploy-a-self-hosted-thirdweb-engine-on-railway-in-less-than-3-minutes).
+
```

Comment:
This example is of self-hosting I believe. Do we need to link it to external blog post?

Reply:
Oh, I'll update it, it was from my original repo but decided to move to our org.

Reply:
Update it with the guide from our support page.